server/src/capture.rs

```diff
@@ -0,0 +1,227 @@
+// src/capture.rs
```

Comment:
Use the CodeChat Editor to provide a title, e.g. ``# `src/capture.rs` - Capture user actions to a database``